_site/index.html

```diff
@@ -0,0 +1,223 @@
+                <div class="box">
+                  <div class="features-img-block"><img src="assets/img/request_blue.svg" class="features-img img-fluid"
+                    alt=""></div>
+                  <h4 class="title">Request & response stubbing</h4>
+                  <p class="description">Mock and stub requests from the browser with 
+                Taiko's <code>intercept</code> API that makes managing test environments and data 
+                simpler.</p>
+                </div>
+              </a>
+            </div>
+          </div>
+
+        </div>
+      </section>
+      <!-- #features -->
+
+    </main>
+
+    <!--==========================
+    Footer
+  ============================-->
+    <footer id="footer" class="section-bg">
+      <div class="footer-top">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-12 footer-content">
+              <div class="row">
+                <div class="col-sm-4">
+                  <div class="footer-links">
+                    <h4>Product</h4>
+                    <ul>
+                      <li>
+                        <a aria-label="Taiko Documentation" href="https://docs.taiko.dev">Documentation</a>
+                      </li>
+                      <li>
+                        <a aria-label="Taiko License" href="https://github.com/getgauge/taiko/blob/master/LICENSE">License</a>
+                      </li>                    
+                      <li>
+                        <a aria-label="Press kit" href="./assets/Taiko_logos.zip">Press kit</a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+                <div class="col-sm-4">
+                  <div class="footer-links">
+                    <h4>Engage</h4>
+                    <ul>
+                      <li>
+                        <a aria-label="Spectrum, forums for support" href="https://spectrum.chat/taiko" class="icon_spectrum">Spectrum</a>
+                      </li>
+                      <li>
+                        <a aria-label="Instructions on how to contribute" href="https://github.com/getgauge/taiko/blob/master/CONTRIBUTING.md" class="icon_contribute">Contribute</a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+                <div class="col-sm-4">
+                  <div>
+                    <a href="https://www.netlify.com">
+                      <img style="width:125px; margin-bottom: 1em" 
+                      src="https://www.netlify.com/img/global/badges/netlify-dark.svg" 
+                      alt="Deploys by Netlify"/>
+                    </a>
+                  </div>
+                  <div class="footer-info">
+                    <img src="./assets/img/taiko_logo.svg" class="footer-logo" alt="taiko">
+                  </div>                  
+                </div>
+              </div>
+            </div>

```

Comment:
This also seems to be the generated file because it is under `_site` 
There is already a version of `index.html` in the root.